CHANGELOG.md

```diff
@@ -7,11 +8,27 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
 
 ## Unreleased
 
+## [2.1.0]
+
 ### Added
-- Support env overrides for ~all command-line flags.
-  - Flags that take multiple values can be repeated on the command line,
-    or passed as comma-separated values via environment or command-line args.
-- Update CI deployments to use zerossl instead of letsencrypt
+
+- Add `Chain-ID` gRPC metadata ([MCIP #49](https://github.com/mobilecoinfoundation/mcips/pull/49)) to provide additional runtime disambiguation between clients and servers.
+- Add a required `--chain-id` command-line arguments to consensus and fog servers.
+- Add an optional `--chain-id` command-line argument to `mobilecoind`.
+- Support env overrides and overlays (for multi-value items) for nearly all command-line arguments.
```

Comment:
Are overlays supported? I think one just take precedence over the other (e.g. if both env var and command line arguments for a given option are present, only the command line arguments will get used)

Reply:
Fixed